lib/datadog/appsec/component.rb

```diff
@@ -28,6 +29,22 @@ def build_appsec_component(settings, telemetry:)
 
         private
 
+        def enable_appsec?(settings)
+          return false unless settings.respond_to?(:appsec) && settings.appsec.enabled
```

Comment:
Do you think inverting `unless &&` makes sense here? I find it hard to understand (negate logic with addition to `&&`)

```suggestion
          return false if !settings.respond_to?(:appsec) || !settings.appsec.enabled
```

Reply:
I agree, this will improve readability

Reply:
For the record I think the `unless` & `&&` version was easier to follow.